src/components/Books.js

```diff
@@ -1,18 +1,24 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import styles from './Books.module.css';
 import AddBook from './AddBook';
 import Book from './Book';
+import { getBooks } from '../redux/books/books';
 
 function Books() {
   // eslint-disable-next-line no-unused-vars
```

Comment:
- [OPTIONAL] Please make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉